csharp/generate_protos.sh

```diff
@@ -28,6 +28,7 @@ fi
 # descriptor.proto and well-known types
 $PROTOC -Isrc --csharp_out=csharp/src/Google.Protobuf \
     --csharp_opt=base_namespace=Google.Protobuf \
+    --csharp_opt=async \
```

Comment:
I believe these can be comma-separated, e.g.

```text
--csharp_opt=base_namespace=Google.Protobuf,async
```

Although I'd be tempted to use "async=true" - I'll need to research what other options do for this.